src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
As stated in the comment, the problem, according to my experiments, is that using the name of a provide instead of the exact package name, a dialog is displayed to the user with a message "libyui-qt6 is going to be installed". Having extra dialogs is quite bad for autoinstallation. When I use the exact package name the dialog doesn't appear.

Is that the case or am I crazy? In case I'm not crazy, any way to avoid the popup while using a provide?